src/background/js/app.js

```diff
@@ -46,6 +62,8 @@ gApp.config(function ($routeProvider, $compileProvider) {
         });
 });
 
+gApp.API_BASE_URL = "http://localhost:8080/api/1/";
```

Comment:
You should probably use the ENV (variable) to check if it's production or development